doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,278 @@
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+
+<p align="center"><b>Table 5-3-6:</b> Controller information.</p>
+
+<a name="5.3.7"></a>
+### 5.3.7 Compute informations
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name(s) which this node assigned to.|
+| manage_vlan_id | String | Yes | may multiple vlan_id for management plane. Setting the dedicated vlan id assigned.|
+| storage_vlan_id | String | Yes | may multiple vlan_id for storage plane. Setting the dedicated vlan id assigned.|
+| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
+| os_reserved_cores | String | No | Designed and planned by provider itself, may be none|
+| ovs_reserved_cores | String | No | Designed and planned by provider itself, may be none |
+
+<p align="center"><b>Table 5-3-7:</b> Compute information.</p>
+
+<a name="5.3.8"></a>
+### 5.3.8 Distributed storage informations
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_infos | List | Yes | a list of cluster information|
+<p align="center"><b>Table 5-3-8-1:</b> Storage information.</p>
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_name | String | Yes | indicate the cluster name|
+| ceph_iscsi_gateway | String | Yes | splitted by semicolon if more than one. e.g 2409:8086:8412:10a::1;2409:8086:8412:10a::2|
+| ceph_manage_plane_address_segment | String | Yes | The address segment of ceph management plane. e.g 2409:8086:8412:100::/64|
+| ceph_storage_plane_address_segment | String | Yes | The address segment of ceph storage plane. e.g 2409:8086:8412:10a::/64|
+| ceph_os_user | String | Yes | the user of ceph OS. |
+| ceph_os_password | String | Yes | the user's password of ceph OS. |
+| ceph_manage_plane_float_ip | String | Yes |  |
```

Comment:
Is the proposal to hardcode CEPH into the descriptor file? Despite Ceph being used all over the place, I don't feel that we should hardcode one particular storage backend, but rather keep information on a generic level, if possible, and let the installer derive the concrete storage backend configuration

Reply:
Thanks gkunz, it makes sense defining a generic one for the storage, I'll update it.

Reply:
Thanks gkunz, it makes sense defining a generic one for the storage, could you share your further advise about the generic abstract?

Reply:
As mentioned in the meeting, I commit a copy of generic definition instead. Please check it @gkunz